doc/article/technical_article.tex

```diff
@@ -111,48 +108,47 @@
 % ------------------------------------------------------------------------------
 \begin{abstract}
   The main goal of the present article is to describe the idea, goals and main
-  functionalities of the web based application trckr. We developed trckr for
+  functionalities of the web-based application trckr. We developed trckr for
   % Since we want trckr to be all lowercase, we avoid using it at the start of
   % the sentence
   everyone who works on a project and needs an intuitive and simple web tool to
   accurately track their time spent on different tasks.
   % Hint: Empty line makes a paragraph which is semantically different from the
   % forced \\ newline (and much neater)
 
-  The backend is written in Python with the help of the web application
-  framework Django and the frontend with the Javascript UI framework Vue.js.
+  The back-end is written in Python with the help of the web application
+  framework Django and the front-end with the Javascript UI framework Vue.js.
   Both technologies were new to most team members, but all members have proven
-  themselves self-driven and learning these skills was ultimately a benefit to
+  themselves self-driven, and learning these skills was ultimately a benefit to
```

Comment:
I'm not sure, but self-driven sounds weird to me when already saying "themselves". Maybe say something like: "...all members had the drive to learn these technologies, which ultimately has proven to be a benefit..."

Reply:
I agree with Gabi